Gopkg.toml

```diff
@@ -0,0 +1,71 @@
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+
+required = [
+  # Requiring a subpackage that contains Go code pulls in the whole
+  # package.
+
+  # For verify/update scripts
+  "k8s.io/code-generator/cmd/client-gen",
+  # For verify scripts.
+  "github.com/kubernetes/repo-infra/verify/boilerplate/test",
+]
+
+ignored = ["github.com/kubernetes/repo-infra/kazel"]
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/go-openapi/spec"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/golang/glog"
+
+[[constraint]]
+  name = "github.com/spf13/cobra"
+  version = "0.0.1"
+
+[[constraint]]
+  name = "github.com/spf13/pflag"
+  version = "1.0.0"
+
+[[constraint]]
+  branch = "master"
+  name = "k8s.io/api"
+
+[[constraint]]
+  branch = "master"
```

Comment:
I would definitely not pin to master of this repo, especially since you have pinned to a specific hash of apiserver.  Likewise with client-go and any other k8s repo.  We will be constantly broken unless we pin to a known set of versions that works well together.

So, I would ensure that we are pinned to known versions of all k8s repos before merging this PR.

Reply:
That makes sense. I had left this like this because the specific version pinning is done in `Gopkg.lock`, and this file will need to be updated regardless when the tool is run again in order to get specific versions at that point.

However, I've updated this accordingly, since I agree that it looks confusing to have `master` in `Gopkg.toml`, and that specific versions maps better to how we actually have to do dependency management for now.